cirq-core/cirq/devices/fidelity.py

```diff
@@ -0,0 +1,216 @@
+from typing import Sequence
+from itertools import product
+import cirq
+import numpy as np
+from cirq.devices.noise_model import validate_all_measurements
+
+
+class Fidelity:
```

Comment:
I'm not sure that this is the right name for this class. It doesn't just store a single fidelity. Perhaps something like `NoiseData` or `NoiseProperties` or `OpFidelities`...